thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessTaskRunner.java

```diff
@@ -153,26 +146,26 @@ private void executeCleanupTask(DataCompletenessTaskInfo dataCompletenessTaskInf
           DataCompletenessConstants.CLEANUP_TIMEUNIT);
       long cleanupOlderThanMillis = new DateTime().minus(cleanupOlderThanDuration).getMillis();
       List<DataCompletenessConfigDTO> findAllByTimeOlderThan =
-          dataCompletenessConfigDAO.findAllByTimeOlderThan(cleanupOlderThanMillis);
+          DC.findAllByTimeOlderThan(cleanupOlderThanMillis);
 
       LOG.info("Deleting {} entries older than {} i.e. {}",
           findAllByTimeOlderThan.size(), cleanupOlderThanMillis, new DateTime(cleanupOlderThanMillis));
       for (DataCompletenessConfigDTO config : findAllByTimeOlderThan) {
-        dataCompletenessConfigDAO.delete(config);
+        DC.delete(config);
       }
 
       // find all entries older than LOOKBACK and still dataComplete=false, mark timedOut
       long timeOutOlderThanDuration = TimeUnit.MILLISECONDS.
           convert(DataCompletenessConstants.LOOKBACK_TIME_DURATION, DataCompletenessConstants.LOOKBACK_TIMEUNIT);
       long timeOutOlderThanMillis = new DateTime().minus(timeOutOlderThanDuration).getMillis();
       List<DataCompletenessConfigDTO> findAllByTimeOlderThanAndStatus =
-          dataCompletenessConfigDAO.findAllByTimeOlderThanAndStatus(timeOutOlderThanMillis, false);
+          DC.findAllByTimeOlderThanAndStatus(timeOutOlderThanMillis, false);
```

Comment:
What DC stands for ? choose meaningful and self explanatory names please.
we have been using xyzDAO for data provider objects.